.golangci.yaml

```diff
@@ -69,6 +72,13 @@ linters-settings:
     ignore-type-assert-ok: true
     ignore-map-index-ok: true
     ignore-chan-recv-ok: true
+  godox:
+    keywords:
+      - TODO
+      - FIXME
```

Comment:
I think TODO and FIX are default. You can either name both of them also here explicitly or only add the additional ones here.